doc/sphinx/user-guide.rst

```diff
@@ -251,32 +219,10 @@ When using a :py:class:`~azstoragetorch.datasets.IterableBlobDataset` and
 :py:class:`~torch.utils.data.DataLoader` with multiple workers (i.e., ``num_workers > 1``), the
 :py:class:`~azstoragetorch.datasets.IterableBlobDataset` automatically shards data samples
 returned across workers to avoid a :py:class:`~torch.utils.data.DataLoader` from returning
-duplicate samples from its workers::
-
-    from azstoragetorch.datasets import IterableBlobDataset
-    from torch.utils.data import DataLoader
-
-    # Update URL with your own Azure Storage account and container name
-    CONTAINER_URL = "https://<my-storage-account-name>.blob.core.windows.net/<my-container-name>"
-
-    dataset = IterableBlobDataset.from_container_url(CONTAINER_URL)
-
-    # Iterate over the dataset to get the number of samples in it
-    num_samples_from_dataset = len([blob["url"] for blob in dataset])
-
-    # Create a DataLoader to load data samples from the dataset in batches of 32 using 4 workers
-    dataloader = DataLoader(dataset, batch_size=32, num_workers=4)
-
-    # Iterate over the DataLoader to get the number of samples returned from it
-    num_samples_from_dataloader = 0
-    for batch in dataloader:
-        num_samples_from_dataloader += len(batch["url"])
+duplicate samples from its workers
 
-    # The number of samples returned from the dataset should be equal to the number of samples
-    # returned from the DataLoader. If the dataset did not handle sharding, the number of samples
-    # returned from the DataLoader would be ``num_workers`` times (i.e., four times) the number
-    # of samples in the dataset.
-    assert num_samples_from_dataset == num_samples_from_dataloader
+    .. literalinclude:: ../../samples/iterable_dataset/multiple_workers.py
+        :lines: 9-36
```

Comment:
I think it is fine to keep the lines at `9-` here. Mainly if someone was to copy and paste it and update the constant, it will work out of the box.